.github/workflows/release.yml

```diff
@@ -13,7 +14,58 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
+  test-kubo:
+    name: 'test:kubo'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: L1-node
+
+      - name: Checkout Kubo fork
```

Comment:
I think this needs a docs/ page, describing how the maintenance of the Kubo fork is done. For example, what are the tasks for keeping it up to date, and how often does one do this?

Reply:
It would also be useful to document how to run these Kubo tests locally, for faster iterations

Reply:
agreed. working on this atm